resources/generate-docs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
@anandkumarpatel manually for now. could be added to a `postpublish` hook w/ `npm`, but I've not looked into those before, so just going one step at a time.